src/browser/main.js

```diff
@@ -842,10 +842,36 @@
             {
                 $("boot_options").style.display = "none";
 
+                parse_param()
                 start_emulation(settings, done);
             }
         }
 
+        function parse_param()
+        {
+            const m = parseInt(query_args["m"], 10);
+            if(m > 0)
+            {
+                settings.memory_size = Math.max(16, m) * 1024 * 1024;
+            }
+
+            const vram = parseInt(query_args["vram"], 10);
+            if(vram > 0)
+            {
+                settings.vga_memory_size = vram * 1024 * 1024;
+            }
+
+            const networking = parseInt(query_args["networking"], 10);
+            settings.networking = networking == 0 ? undefined : networking;
+
+            const audio = !!parseInt(query_args["audio"], 10);
```

Comment:
I believe this should be called `disable_audio`. Alternatively, if you want to keep the name `audio` (which is a bit nicer), it should default to true.